app/controllers/payments_controller.rb

```diff
@@ -13,12 +13,16 @@ def new
 
   end
 
-  # Disable
-=begin
+
   def edit
```

Comment:
@gsm99  Need to add in access layer to prevent merchants from editing others' payments